<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OracleAuthDlg</class>
 <widget class="QDialog" name="OracleAuthDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация Oracle</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/fmt.qrc">
    <normaloff>:/db</normaloff>:/db</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QTreeView" name="connectionsTree">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Аутентификация</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Пользователь:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_username">
              <property name="text">
               <string>SERP_3174</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Пароль:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_passw">
              <property name="text">
               <string>SERP_3174</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>База данных:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBox_Dsn">
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="currentText">
               <string>THOR_DB12DEV1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>108</width>
           <height>71</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="delRecentButton">
       <property name="text">
        <string>Удалить закладку</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_accept">
       <property name="text">
        <string>Подключиться</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButto_close">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_passw</tabstop>
  <tabstop>pushButton_accept</tabstop>
  <tabstop>pushButto_close</tabstop>
 </tabstops>
 <resources>
  <include location="../res/fmt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButto_close</sender>
   <signal>clicked()</signal>
   <receiver>OracleAuthDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
